src/current/v23.2/cluster-virtualization-overview.md

```diff
@@ -0,0 +1,104 @@
+
+- The system interface manages the cluster's control plane. Administrative access to the system interface can be restricted. Certain low-level cluster settings can be modified only on the system interface.
+- One or more virtual clusters manage their own data plane. Administrative access on a virtual cluster does not grant any access to the system interface. The effect of some settings is scoped to the virtual cluster rather than the system interface.
+
+## Differences when cluster virtualization is enabled
+
+When cluster virtualization is enabled, CockroachDB's behavior changes in several key areas, as described in the following sections. For details about working with a cluster with cluster virtualization enabled, refer to [Work with virtual clusters]({% link {{ page.version.version }}/work-with-virtual-clusters.md %}).
+
+### Connecting to a cluster
+
+When cluster virtualization is enabled, by default when you connect using `cockroach sql` or the DB Console, you are connected to the virtual cluster. To connect to the system interface, you set `-ccluster=system` in the connection string (for SQL clients) or the DB Console URL. For details, refer to [Work with virtual clusters]({% link {{ page.version.version }}/work-with-virtual-clusters.md %}#connecting-to-a-cluster).
+
+{{site.data.alerts.callout_success}}
+If a SQL user has been added to the system interface and one or more virtual clusters with the same password, that user can select which to connect to from the top of the DB Console.
+{{site.data.alerts.end}}
+
+### Cluster settings
+
+When [cluster virtualization]({% link {{ page.version.version }}/cluster-virtualization-overview.md %}) is enabled, each [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) has a scope, which may be the virtual cluster or the system interface.
+
+- When a cluster setting is scoped to the virtual cluster, it affects only the virtual cluster and not the system interface. To configure a cluster setting that is scoped to the virtual cluster, you must have the `admin` role on the virtual cluster, and you must connect to the virtual cluster before configuring the setting. The majority of cluster settings are scoped to the virtual cluster and are visible only when connected to the virtual cluster.
+- When a cluster setting is scoped to the system interface, it effects the entire storage cluster. To configure a cluster setting that is scoped to the system interface, you must have the `admin` role on the system interface, and you must connect to the system interface before configuring the setting.
+- When a cluster setting is system-visible, it can be set only from the system interface but can be queried from any virtual cluster. For example, virtual cluster can query a system-visible cluster setting's value to help adapt to the storage cluster's configuration.
```

Comment:
```suggestion
- When a cluster setting is system-visible, it can be set only from the system interface but can be queried from any virtual cluster. For example, a virtual cluster can query a system-visible cluster setting's value to help adapt to the storage cluster's configuration.
```

It might also be helpful to provide a real example here.